### PR TITLE
VACMS_3265: Format situation update time field widget.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #3265 

## Testing done
Visual

## Screenshots
image

## QA steps
Kev to do this one (please)

### CODE REVIEW

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Looks good Ethan.   One minor and one moderate request.

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 1]
```diff
@@ -622,12 +637,15 @@ function va_gov_backend_field_widget_paragraphs_form_alter(&$element, FormStateI
    * @var \Drupal\field\Entity\FieldConfig $field_definition */
   $paragraph_entity_reference_field_name = $field_definition->getName();
 
-  if ($paragraph_entity_reference_field_name === 'field_content_block' || $paragraph_entity_reference_field_name === 'field_service_location') {
+  if ($paragraph_entity_reference_field_name === 'field_content_block' || $paragraph_entity_reference_field_name === 'field_service_location' || $paragraph_entity_reference_field_name === 'field_situation_updates') {
```
[the remark this answers is not in the document]
Comment (ethanteague) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module:640]: I think the line limit thing is maybe set to not trigger a phpcs violation in our container settings? I'll refactor this, @swirtSJW :+1:

[the remark this answers is not in the document]
Comment (swirtSJW) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module]: Nice.  Thanks.

**Review by swirtSJW — APPROVED** [review]

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
QA steps  I am not sure what to do with this one.  @kevwalsh can I merge?

**Comment by ethanteague** [comment]
@kevwalsh can you qa this one?

**Comment by kevwalsh** [comment]
Sorry for delay on the QA!
Edit_VAMC_System_Banner_Alert_with_Situation_Updates_Help_Limit_the_Spread_of_COVID-19_and_other_Illnesses___VA_gov_CMS

**Comment by ethanteague** [comment]
@swirtSJW - looks like we are good to merge this one in.

**Pull request merged**